install.sh

```diff
@@ -83,20 +85,22 @@ function downloadConfig() {
     local artifact="config.toml"
     local url="${base}/${VERSION}/${artifact}"
 
-    curl -SsL "${url}" -o "${artifact}"
+    if ! curl --silent --location --fail "${url}" -o "${artifact}"; then
+        exitWithErr "Config download failed"
+    fi
 }
 
 function verifyCLI() {
-    if command -v "./${APP}" > /dev/null; then
+    if ./${APP} --help > /dev/null 2>&1; then
```

Comment:
`command -v` will always succeed due to `chmod +x "${APP}"` in the downloadCLI step. updated to use the `--help` command as a stand-in for a working binary